core/src/Xios.cpp

```diff
@@ -32,31 +32,29 @@
 
 namespace Nextsim {
 
-static const std::map<int, std::string> keyMap = { { Xios::ENABLED_KEY, "xios.enable" } };
+static const std::map<int, std::string> keyMap
+    = { { Xios::ENABLED_KEY, "xios.enable" }, { Xios::START_TIME_KEY, "model.start" },
+          { Xios::TIME_STEP_KEY, "model.time_step" }, { Xios::PERIOD_KEY, "ConfigOutput.period" } };
```

Comment:
Use a different configuration section to configure XIOS output.

Reply:
Addressed in 89e37647dc13cbc9d02ba86620bb4841cb51d880.